satpy/tests/test_crefl_utils.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+
+class testCreflUtils(unittest.TestCase):
```

Comment:
Since your my student @wroberts4 I'm going to ask you to fix the name of this class and the name of your test functions to be PEP8 compatible. `TestCreflUtils` and `test_get_atm_variables_abi`

Reply:
Try to review other tests that you've added.